Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2635,6 +2637,41 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     }
 }
 
+void WebExtensionContext::compileContentModificationRules(NSArray *rulesData)
```

Comment:
Saying declarativeNetRequest is fine internally, if you want. I just wanted the public API to be generic. But this is fine too!

Reply:
I'm going to be bringing in mode code with the declarativeNetRequest naming. I'll change this to be more specific.